pkg/querier/tripperware/instantquery/instant_query.go

```diff
@@ -203,6 +203,9 @@ func (instantQueryCodec) EncodeRequest(ctx context.Context, r tripperware.Reques
 		}
 	}
 
+	// Always ask gzip to the querier
+	h.Set("Accept-Encoding", "gzip")
```

Comment:
If we configure the forwarded headers with `Accept-Encoding` here https://github.com/cortexproject/cortex/blob/master/pkg/querier/tripperware/roundtrip.go#L120 then is this still necessary?

Reply:
If we configured that before it would break everything.. as the result would be compressed and we were not expecting that! =/

We wanna send always.. not only when the customer send..